test/Microsoft.AspNetCore.Server.KestrelTests/FrameResponseHeadersTests.cs

```diff
@@ -14,73 +14,6 @@ namespace Microsoft.AspNetCore.Server.KestrelTests
     public class FrameResponseHeadersTests
     {
         [Theory]
-        [InlineData(true)]
-        [InlineData(false)]
-        public void InitialDictionaryContainsServerAndDate(bool addServerHeader)
```

Comment:
I'd rather have this converted into a test that checks those headers are *not* there.